app/src/main/java/com/example/paulsuarez/a27_facebook_lite/Friend.java

```diff
@@ -0,0 +1,12 @@
+package com.example.paulsuarez.a27_facebook_lite;
```

Comment:
Oh, one tip I forgot to mention: From an organizational standpoint, it's a good idea to create a models subfolder and put model files like this one in that folder.